### PR TITLE
chore: convert to astro

### LINKED ISSUE #9: [CHORE] - Port to Astro and Netlify
### What work needs to be performed?

The site needs to be ported to use Astro as the framework and deployed via netlify.

### Additional information

Someone will need to use the bubble bot to create and deploy the site on Netlify

### PR DESCRIPTION
**Pull request by kcoderhtml**
# Merge Astro migration branch

## Description:

Converted everything to Astro and added netlify.toml

## Related Issue:
Closes #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Astro Starter Kit: Basics
```
Comment (jdogcoder) [inline README.md]: At some point we should probably customize this for the site, looks okay for now tho!

File: package.json [hunk 2]
```diff
@@ -1,23 +1,15 @@
 {
-  "name": "thepurplebubble-site",
-  "version": "1.0.0",
-  "private": true,
-  "dependencies": {
-    "@lottiefiles/lottie-player": "1.5.7",
-    "parcel": "^2.10.3"
-  },
+  "name": "",
```
Comment (jdogcoder) [inline package.json:2]: Can we add our name back?

File: package.json [hunk 3]
```diff
@@ -1,23 +1,15 @@
 {
-  "name": "thepurplebubble-site",
-  "version": "1.0.0",
-  "private": true,
```
Comment (jdogcoder) [inline package.json:4]: We can keep   "private": true, in

File: package.json [hunk 4]
```diff
@@ -1,23 +1,15 @@
...
-  "dependencies": {
-    "@lottiefiles/lottie-player": "1.5.7",
-    "parcel": "^2.10.3"
-  },
+  "name": "",
+  "type": "module",
+  "version": "0.0.1",
   "scripts": {
-    "prepare": "husky install"
```
Comment (jdogcoder) [inline package.json:10]: This should stay as part of the linting system!

File: package.json [hunk 5]
```diff
@@ -1,23 +1,15 @@
...
-    "lint-staged": "^15.2.0",
-    "prettier": "3.1.1",
-    "prettier-eslint": "^16.1.2"
-  },
-  "lint-staged": {
-    "**/*": "prettier --write --ignore-unknown"
+  "dependencies": {
+    "astro": "^4.0.4"
   }
-}
```
Comment (jdogcoder) [inline package.json:23]: This should also stay as part of the linting system!

**Comment by jdogcoder** [comment]
@kcoderhtml this good to re review?

**Comment by kcoderhtml** [comment]
yup, should be good to go!

**Review by jdogcoder — APPROVED** [review]

**Pull request merged**